requirements.txt

```diff
@@ -1,36 +1,26 @@
-altgraph==0.17
-amply==0.1.4
-cycler==0.10.0
-decorator==4.4.2
-docutils==0.17.1
-future==0.18.2
+Pillow==9.3.0
+PuLP==2.7.0
+PyQt5==5.15.7
+PyQt5-Qt5==5.15.2
+PyQt5-sip==12.11.0
+PyQt5-stubs==5.15.6.0
+XlsxWriter==3.0.3
+contourpy==1.0.6
+cycler==0.11.0
+fonttools==4.38.0
 grandalf==0.7
 grinpy==19.7a0
-importlib-metadata==4.11.3
-kiwisolver==1.4.2
-macholib==1.14
-matplotlib==3.4.2
-modulegraph==0.18
-netgraph==4.4.1
+kiwisolver==1.4.4
+matplotlib==3.6.2
+netgraph==4.10.2
 network2tikz==0.1.8
-networkx==2.5.1
-numpy==1.21.6
-pefile==2021.5.13
-Pillow==9.1.1
-PuLP==2.6.0
-pyinstaller==5.0.1
-pyinstaller-hooks-contrib==2022.5
-pyparsing==2.4.7
-PyQt5==5.15.4
-PyQt5-Qt5==5.15.2
-PyQt5-sip==12.9.0
-PyQt5-stubs==5.15.2.0
+networkx==2.8.8
+numpy==1.23.4
+packaging==21.3
+pyparsing==3.0.9
 python-dateutil==2.8.2
-pywin32-ctypes==0.2.0
 rectangle-packer==2.0.1
-scipy==1.7.1
-simpleeval==0.9.11
+scipy==1.9.3
+simpleeval==0.9.12
 six==1.16.0
-typing-extensions==4.2.0
-XlsxWriter==3.0.3
```

Comment:
Are you sure this is not being used? Which library are we using to generate the Excel document? 🤔

Reply:
But we are using XlsxWriter, it was just moved.

https://github.com/GraphFilter/GraphFilter/blob/72c4c4a2d11f9b8c846c8c3d03e6024a11b25a11/requirements.txt#L7

Reply:
You are right! My apologies.